src/bootstrap/src/utils/tarball.rs

```diff
@@ -64,6 +65,9 @@ impl OverlayKind {
                 "compiler/rustc_codegen_cranelift/LICENSE-APACHE",
                 "compiler/rustc_codegen_cranelift/LICENSE-MIT",
             ],
+            OverlayKind::LlvmBitcodeLinker => {
```

Comment:
Why is this a separate/new overlay kind rather than just using OverlayKind::Rust?